reservation/Dockerfile

```
@@ -0,0 +1,32 @@
+WORKDIR /app
+
+COPY gradlew .
+COPY gradle gradle
+COPY authorization-shared authorization-shared
+COPY reservation reservation
+
+RUN echo "include 'reservation'" > settings.gradle && \
+    echo "include 'authorization-shared'" >> settings.gradle
+
+RUN chmod +x gradlew
+RUN ./gradlew :reservation:clean :reservation:build -x test --no-daemon
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)